collectors/metadata/schemas/single-module.json

```
@@ -11,13 +11,45 @@
       "type": "string",
       "description": "Module name (e.g. apache, /proc/stat, httpcheck). It usually has the same name as the module configuration file (external plugin) or the section name in netdata.conf (internal plugin)."
     },
+    "categories": {
+      "type": "array",
+      "description": "An array defining the categories the integration falls into.",
+      "items": {
+        "$ref": "#/$defs/_categories"
```

```suggestion
        "$ref": "#/$defs/_categories"
```

Let's rename it to `_category` because it is a category (one entity, not multiple).

---

true that